### PR TITLE
gopass: update to 1.13.0

### PR DESCRIPTION
**Pull request by dominikschulz**
gopass: update to 1.13.0

### CODE REVIEW

File: srcpkgs/gopass/template [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Template file for 'gopass'
 pkgname=gopass
-version=1.12.8
-revision=1
```
Comment (paper42) [inline srcpkgs/gopass/template:4]: revision is always necessary, don't remove it

**Review by paper42 — COMMENTED** [review]
please fill out the PR template

**Comment by dominikschulz** [comment]
This is an auto-generated change as part of our release process. I don't see anything in the template that would apply here.

**Comment by paper42** [comment]
> This is an auto-generated change as part of our release process. I don't see anything in the template that would apply here.

in that case
```
#### Testing the changes
- I tested the changes in this PR: **NO**
```

**Pull request merged**